src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingConfigSourceProvider.java

```diff
@@ -12,30 +13,37 @@
 import org.eclipse.microprofile.config.spi.ConfigSourceProvider;
 
 /**
- * Loading the YAML configuration file from the resource folder or env variable and making the
- * config available to the application.
+ * Loading the YAML configuration file from the resource folder or file path and making the config
+ * available to the application.
+ *
+ * <ol>
+ *   <li>With env {@link EmbeddingConfigSourceProvider#EMBEDDING_CONFIG_PATH},{@link
+ *       EmbeddingConfigSourceProvider#RERANKING_CONFIG_PATH} variable set, Data API loads provider
+ *       config from specified file location. E.G. Astra Data API.
+ *   <li>With system property {@link EmbeddingConfigSourceProvider#DATA_API_INTEGRATION_TEST} set,
+ *       Data API loads test provider config resource. E.G. Data API integration test.
+ *   <li>With none set, Data API loads default provider config from resource folder. E.G. Local
+ *       development.
+ * </ol>
  */
 @StaticInitSafe
 public class EmbeddingConfigSourceProvider implements ConfigSourceProvider {
-  private static final String EMBEDDING_CONFIG_ENV = "EMBEDDING_CONFIG_PATH";
-  private static final String RERANKING_CONFIG_ENV = "RERANKING_CONFIG_PATH";
-  private static final String EMBEDDING_CONFIG_RESOURCE = "embedding-providers-config.yaml";
-  private static final String RERANKING_CONFIG_RESOURCE = "reranking-providers-config.yaml";
+  private static final String EMBEDDING_CONFIG_PATH = "EMBEDDING_CONFIG_PATH";
```

Comment:
Could use Javadoc -- this is the ENV variable name to use for path to embedding config document?